recipes/onmail/webview.js

```diff
@@ -0,0 +1,22 @@
+const _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
+
+module.exports = Ferdium => {
+  const getMessages = () => {
+    let countImportant = 0;
+    const inboxLinks = document.querySelectorAll('p.truncate');
+    for (const label of inboxLinks){
+      if (label.textContent) {
```

Comment:
In the if condition, you are checking for `label.textContent`, but the next line, you are using `label.nextSibling.textContent` - is that correct?

Reply:
yes it is, the p is a sibling of the actual version, the p.truncate are a lot of labels, the only one we care about is the `inbox` one, so instead of defaulting to the first node, I decided to loop them and check, I think that is a better solution, in case the order changes or they add a new node before or what not.

Reply:
since i dont have this service login, was just making sure that this was not an oversight. Thanks